tools/project_manager/editproject.php

```diff
@@ -742,23 +747,25 @@ function save_to_db()
             }
 
             // Insert a new row into the projects table
-            mysqli_query(DPDatabase::get_connection(), "
+            $sql = sprintf("
                 INSERT INTO projects
                 SET
-                    projectid    = '{$this->projectid}',
+                    projectid    = '%s',
                     $pm_setter
                     state        = '".PROJ_NEW."',
```

Comment:
For consistency, we should move the state into the `sprintf()` replace too.

Reply:
PHP! :(